Reflection/Chat.simba

```diff
@@ -151,6 +151,14 @@ begin
     DialogContinue.Free;
     Exit(True);
   end;
+
+  NewActionsContinue := RSWidget.Get(R_INTERFACE_NEW_ACTIONS_CONTINUE.Group, R_INTERFACE_NEW_ACTIONS_CONTINUE.Child);
+  if NewActionsContinue.ref <> nil then
```

Comment:
Could we potentially optimize this check? For example, if the NPCContinue is not found, then check for this, otherwise, just deal with the NPCContinue?

Reply:
ah yeah, definitely. I'll make that change.

Reply:
refactored it to just loop through, potentially clicking any that are found (e.g. when you level up you sometimes need to click through 2 of them)